snippets/csharp/System.Linq/Queryable/AggregateTSource/queryable.cs

```diff
@@ -1398,10 +1415,10 @@ static void LastOrDefaultEx3()
         public static void LongCountEx1()
         {
             // <Snippet47>
-            string[] fruits = { "apple", "banana", "mango",
-                                  "orange", "passionfruit", "grape" };
+            string[] fruits = [ "apple", "banana", "mango",
+                                  "orange", "passionfruit", "grape" ];
 
-            long count = fruits.AsQueryable().LongCount();
+            long count = fruits.LongCount();        // [CA1829, RCS1077] better to use "fruits.Length;"
```

Comment:
I'll stop commenting on this point, but all these should be removed when the purpose is to show the given extension method:

```suggestion
            long count = fruits.LongCount(); 
```